langlib/lang.object/src/main/ballerina/object.bal

```diff
@@ -22,6 +22,6 @@ public type Iterable distinct object {
     #
     # + return - a new iterator object
     public function iterator() returns object {
-        public isolated function next() returns record {| any|error value; |}|error?;
+        public function next() returns record {| any|error value; |}|error?;
```

Comment:
This change has been done on the alpha4-stage branch, wonder if we should wait for that branch to be merged first, to avoid conflicts?

Reply:
this change is from cherry-picking commit from the stage branch. Hence, it will not conflict

Reply:
Ah yes, no problem then.